pages/clustering/high-availability.mdx

```diff
@@ -369,6 +380,14 @@ RPC request deletes all current data on all databases on a given instance and ac
 From follower coordinators you can only execute `SHOW INSTANCES`. Registration of data instance, unregistration of data instances, demoting instance, setting instance to MAIN and force reseting cluster state
 are all disabled. 
 
+<Callout type="warning">
+
+It is important to note that our current implementation of high availability could lead to data loss. The specific situation could occur when all REPLICAs are behind the current MAIN
+and current MAIN is down. The failover is triggered in which the most-up-to-date replica is chosen for promotion. Such an incident can only happen in high-volume environment where
+data is constantly changed, added, deleted etc. If zero data-loss HA is your requirement, please reach us and we will work towards improving that.
```

Comment:
Reach out where? Maybe change to: "please open an issue on GitHub (+link) and ..."